src/chapter01/exercises/exercise02.lean

```diff
@@ -9,13 +10,74 @@ def E : set ℝ := {1}
 -- Which pair of these sets has the property that neither is contained in the other?
 lemma parta : ∃ S T ∈ ({B, C, D, E} : set (set ℝ)), ¬ (S ⊆ T) ∧ ¬ (T ⊆ S) :=
 begin
-  sorry
+  have h0 : D = ({0} : set ℝ),
+  { simp_rw D,
+    ext,
+    split,
+    { intro hx,
+      simp at hx,
+      simp,
```

Comment:
Don't do non-terminal `simp`s. Use `squeeze_simp` to find out the name of the lemma or lemmas being used, and then rewrite them explicitly.